diff --git a/node_modules/vm-browserify/index.js b/node_modules/vm-browserify/index.js
index 4292de7..be8b4b0 100644
--- a/node_modules/vm-browserify/index.js
+++ b/node_modules/vm-browserify/index.js
@@ -107,7 +107,7 @@ Script.prototype.runInContext = function (context) {
 };
 
 Script.prototype.runInThisContext = function () {
-    return eval(this.code); // maybe...
+    return (0, eval)(this.code); // maybe...
 };
 
 Script.prototype.runInNewContext = function (context) {
